library/src/main/res/layout/exo_playback_control_view.xml

```diff
@@ -58,7 +58,7 @@
     <TextView android:id="@id/exo_position"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
-      android:textSize="14sp"
+      android:textSize="13sp"
```

Comment:
I don't think we should be changing our code for this case. Making the font the right size for custom ROMs at the cost of making it too small for the 99.9% case is not acceptable. I'd hypothesize that fonts that cause this issue are not sized correctly, although this is just a guess.

As an aside, the sizes used here are just taken from the Android platform playback controls view, so any app using the platform provided controls presumably encounters the same issue when using fonts in this way.

Happy to merge the other parts of this pull request, but please revert the modified font size part.